doc/faq.rst

```diff
@@ -100,12 +100,37 @@ See :ref:`adding_graphical_models`.
 Will you add GPU support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-No, or at least not in the near future. The main reason is that GPU support
-will introduce many software dependencies and introduce platform specific
-issues. scikit-learn is designed to be easy to install on a wide variety of
-platforms. Outside of neural networks, GPUs don't play a large role in machine
-learning today, and much larger gains in speed can often be achieved by a
-careful choice of algorithms.
+Adding GPU support by default would introduce heavy harware-specific software
+dependencies which make it both harder for the average user to install
+scikit-learn and harder for the developers to maintain the code.
+
+However, since 2023, a limited but growing :ref:`list of scikit-learn
+estimators <array_api_supported>` can already run on GPUs if the input data is
+provided as a PyTorch or CuPy array and if scikit-learn has been configured to
+accept such inputs as explained in :ref:`array_api`. This Array API support
+allows scikit-learn to run on GPUs without introducing heavy and
+hardware-specific software dependencies to the main package.
+
+Most estimators that rely on NumPy for their computationally intensive operations
+can be considered for Array API support and therefore GPU support.
+
+However, not all scikit-learn estimators are amenable to efficiently running
+on GPUs via the Array API for fundamental algorithmic reasons. For instance,
+tree-based models currently implemented with Cython in scikit-learn are
+fundamentally not array-based algorithms. Other algorithms such as k means or k
+nearest neighbors rely on array-based algorithms but are also implemented in
+Cython in order to manually interleave consecutive array operations to avoid
+introducing performance killing memory access to large intermediate arrays:
```

Comment:
Can this sentence be simplified or broken down?

Reply:
Done.